src/core/database/database.service.ts

```diff
@@ -567,35 +567,34 @@ export class DatabaseService {
     aclEditProp,
   }: {
     session: ISession;
-    input: { [Key in keyof TObject]?: UnwrapSecured<TObject[Key]> };
+    input: { [Key in keyof TObject]?: any };
```

Comment:
`id` could be pulled out explicitly here

Reply:
Do you mean update `createNode` to take a separate `id` argument? Or destructure `input` so that `id` is pulled out of the argument?

Reply:
destructure so you don't have to have the if condition in the loop

Reply:
Updated!